### ISSUE TITLE
Markdown rewrites linked image urls as anchor for the image 

### ISSUE THREAD

**Issue by Tricky1975** [gamejolt/issue-tracker#963]
I've seen a few forum posts of mine where I used images hotlinked from ImgUr.com (screenshots on the Screenshot showcase).
It seemed though those images got transferred somehow onto the Game Jolt server (I never uploaded them there). Now I did make the images linkable to a bigger picture to allow people to view details as on the forums pictures (screenshots in particular) can be a bit obscured due to resizing. But now my magnification links do not magnify the picture, in stead they force the browser to download them, and that was NOT my intension.

**Comment by YLivay** [comment]
Can you give an example of the markdown youd write to get the mafnification? Theres actually not much difference between downloading and viewing an image in the browser, its kind of up to  the browser's implementation so it sounds like you should be doing it like that anyways S:

**Comment by Tricky1975** [comment]
First of all [this forum post](https://gamejolt.com/x/permalink/forum-post/352546) is the post that made me discover this issue.

The original markdown I wrote to cause that magnification effect was:
~~~markdown
[![](https://i.imgur.com/cXoZj5B.png)(https://i.imgur.com/cXoZj5B.png)
~~~

As the forums automatically resize pictures (for reasons I fully support) I write this code to make magnifications possible. The GJ system did replace both URLs, so the magnified picture, is now also read from GJ and appears to be causing a forced download (I tried it. On my system it still happens in both Google Chrome and Opera).
